src/ResourceManager/Compute/Commands.Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -112,141 +114,155 @@ public partial class NewAzureRmVmss : ComputeAutomationBaseCmdlet
         [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
         public int[] NatBackendPort { get; set; }
 
-        async Task SimpleParameterSetExecuteCmdlet(IAsyncCmdlet asyncCmdlet)
-        {
-            ResourceGroupName = ResourceGroupName ?? VMScaleSetName;
-            VirtualNetworkName = VirtualNetworkName ?? VMScaleSetName;
-            SubnetName = SubnetName ?? VMScaleSetName;
-            PublicIpAddressName = PublicIpAddressName ?? VMScaleSetName;
-            SecurityGroupName = SecurityGroupName ?? VMScaleSetName;
-            LoadBalancerName = LoadBalancerName ?? VMScaleSetName;
-            FrontendPoolName = FrontendPoolName ?? VMScaleSetName;
-            BackendPoolName = BackendPoolName ?? VMScaleSetName;
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int[] DataDiskSizeGb { get; set; }
```

Comment:
DataDiskSizeInGb

Reply:
Is there a good ValidateRange for this?  a minimum and a maximum?

Reply:
I'm not aware of validation rules. I assume, it can't be negative. How to define validation rules for arrays?